productivity-suite/piercing-library/src/message-bus/get-bus.ts

```diff
@@ -21,9 +25,16 @@ export function getBus(element?: Element): MessageBus {
   if (hasMessageBus(root)) {
     return root[messageBusProp];
   }
-  throw new Error("No global message bus defined!");
+  console.warn("No global message bus defined - creating an empty one.");
+
+  return ((globalThis as unknown as { [messageBusProp]?: GenericMessageBus })[
+    messageBusProp
+  ] = new GenericMessageBus());
 }
 
+/**
+ * Is the given `obj` a `MessageBus` container?
+ */
```

Comment:
I am not too sure if this comment is adding much 😕

Reply:
I added another line!